getting-started/windows-calico/quickstart.md

```diff
@@ -56,25 +57,153 @@ calicoctl ipam configure --strictaffinity=true
 
 The following steps install a Kubernetes cluster on a single Windows node, with a Linux control node.
 
-- **Kubernetes**
+- **Kubernetes VXLAN**
   
   The geeky details of what you get by default:
-  {% include geek-details.html details='Policy:Calico,IPAM:Calico,CNI:Calico,Overlay:VXLAN,Routing:BGP,Datastore:Kubernetes' %}
+  {% include geek-details.html details='Policy:Calico,IPAM:Calico,CNI:Calico,Overlay:VXLAN,Routing:Calico,Datastore:Kubernetes' %}
+
+- **Kubernetes BGP**
+  
+  The geeky details of what you get by default:
+  {% include geek-details.html details='Policy:Calico,IPAM:Calico,CNI:Calico,Overlay:No,Routing:BGP,Datastore:Kubernetes' %}
 
 - **EKS**
 
   The geeky details of what you get by default:
   {% include geek-details.html details='Policy:Calico,IPAM:AWS,CNI:AWS,Overlay:No,Routing:VPC Native,Datastore:Kubernetes' %}
 
 {% tabs %}
-  <label:Kubernetes,active:true>
+  <label:Kubernetes VXLAN,active:true>
   <%
-  
-1. Disable BGP since we are using VXLAN:
+
+1. Ensure that BGP is disabled.
+
    ```bash
    kubectl patch installation default --type=merge -p '{"spec": {"calicoNetwork": {"bgp": "Disabled"}}}'
    ```
+  
+1. Prepare directory for Kubernetes files on Windows node.
+
+   ```powershell
+   mkdir c:\k
+   ```
+
+1. Copy the Kubernetes kubeconfig file from the master node (default, Location $HOME/.kube/config), to **c:\k\config**.
+
+1. Download the powershell script, **install-calico-windows.ps1**.
+
+   ```powershell
+   Invoke-WebRequest {{ "/scripts/install-calico-windows.ps1" | absolute_url }} -OutFile c:\install-calico-windows.ps1
+   ```
+
+1. Run install-calico-windows.ps1 with correct parameters. The powershell script will perform following tasks.
```

Comment:
This wording in this step is a bit confusing. Suggest the following for each platform:

Install Calico for Windows for your datastore with using the default parameters or [customize installation parameters]. (#configure-installation-parameters).
The powershell script downloads Calico for Windows release binary, Kubernetes binaries, Windows utilities files, configures Calico for Windows, and starts the Calico service.
(Delete this next line...not needed)
You do not need to pass a parameter if the default value of the parameter is correct for you cluster.

Reply:
Good point. Doc updated.